packages/react-form/src/hooks/field/field.ts

```diff
@@ -188,9 +194,17 @@ export function useField<Value = string>(
 function normalizeFieldConfig<Value>(
   input: FieldConfig<Value> | Value,
 ): FieldConfig<Value> {
-  if (input == null || typeof input !== 'object') {
-    return {value: input, validates: () => undefined};
+  if (isFieldConfig(input)) {
+    return input;
   }
 
-  return input as FieldConfig<Value>;
+  return {value: input, validates: () => undefined};
+}
+
+function isFieldConfig<Value>(input: any): input is FieldConfig<Value> {
+  return (
+    typeof input === 'object' &&
```

Comment:
`input` can't be null, right?

Reply:
nope, at least not if they are using TS :P